test/e2e/serial/tests/workload_overhead.go

```diff
@@ -274,6 +274,158 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload overhea
 				}
 
 			})
+
+			It("[test_id:47582][tier2] Pod pending when resources requested + pod overhead don't fit on the target node; NRT objects are not updated", func() {
+				var targetNodeName string
+				var targetNrtInitial *nrtv1alpha1.NodeResourceTopology
+				var targetNrtListInitial nrtv1alpha1.NodeResourceTopologyList
+				var err error
+
+				// even if it is not a hard rule, and even if there are a LOT of edge cases, a good starting point is usually
+				// in the ballpark of 5x the base load. We start like this
+				podResources := corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("6"),
+					corev1.ResourceMemory: resource.MustParse("6Gi"),
+				}
+
+				// to avoid issues with fractional resources being unaccounted atm, we round up to requests;
+				// for the test proper, as low as cpu=100m and mem=100Mi would have been sufficient.
+				minRes := corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1"),
+					corev1.ResourceMemory: resource.MustParse("1Gi"),
+				}
+
+				// need a zone with resources for overhead, pod and a little bit more to avoid zone saturation
+				klog.Infof("kubernetes pod fixed overhead: %s", e2ereslist.ToString(rtClass.Overhead.PodFixed))
+
+				const requiredNUMAZones = 2
+				By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+				nrtCandidates := e2enrt.FilterZoneCountEqual(nrts, requiredNUMAZones)
+
+				const neededNodes = 1
+				if len(nrtCandidates) < neededNodes {
+					Skip(fmt.Sprintf("not enough nodes with at least %d NUMA Zones: found %d, needed %d", requiredNUMAZones, len(nrtCandidates), neededNodes))
+				}
+				candidateNodeNames := e2enrt.AccumulateNames(nrtCandidates)
+
+				var ok bool
+				targetNodeName, ok = e2efixture.PopNodeName(candidateNodeNames)
+				Expect(ok).To(BeTrue(), "cannot select a node among %#v", candidateNodeNames.List())
+				By(fmt.Sprintf("selecting node to schedule the test pod: %q", targetNodeName))
+
+				err = fxt.Client.List(context.TODO(), &targetNrtListInitial)
+				Expect(err).ToNot(HaveOccurred())
+				targetNrtInitial, err = e2enrt.FindFromList(targetNrtListInitial.Items, targetNodeName)
+				Expect(err).NotTo(HaveOccurred())
+
+				//get maximum available node CPU and Memory
```

Comment:
If the pod is asking for maximum available node resources, the target node would be disqualified by the `NodeResourceFit` plugin which is prior to the `TopologyMatch` (our) plugin.
IOW,  we're testing core K8S functionality here.

What we need to do is to make sure that the target node considers a good candidate by the `NodeResourceFit` plugin, but not by the `TopologyMatch`.
We can ask for maximum available resources at the zone level for example.

Reply:
Sure, the description in the test plan made me take this approach but what you are suggesting makes sense and I will update shortly.